### DISCUSSION TITLE
myviolet

### DISCUSSION THREAD

**Discussion by Shigoru77** [ylabonte/ioBroker.procon-ip#96] [Q&A] [answered]
Hello,
does this adapter also works for the newer hardware of the same company - violet?

This one has also a API as well
https://www.pooldigital.de/poolsteuerungen/violet-poolsteuerung/

BR

**Comment by ylabonte** [comment, marked as answer]
Hello,
the short answer to your question is: No. 

The Violet API has nothing in common with the ProCon.IP API. It would be easier to write a completely new adapter than to extend this one in such a way that it supports the Violet. That is why there will be no such update in the future. Sorry.
